### DISCUSSION TITLE
Is there a way to display slide number on every slide?

### DISCUSSION THREAD

**Discussion by fanurs** [slidevjs/slidev#351] [Q&A] [answered]
Hi,

First of all, thanks for the wonderful project. I've just found out Slidev yesterday, and I am now starting to learn it. Hopefully, I can create something by the coming Monday -- I have a presentation!

Coming from a background with very little web dev experience, I'm not quite sure how can I display the slide number on each slide, say, on the bottom right corner.

I do realize that on the [navigation bar](https://sli.dev/guide/navigation.html#navigation-bar), there is already the slide number and the total number of slides. Perhaps there is a way for me extract those numbers?

Thanks,
Fanurs

**Comment by antfu** [comment, marked as answer]
https://sli.dev/custom/vue-context.html + https://sli.dev/custom/global-layers.html
